bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```diff
@@ -81,6 +81,19 @@ numConversionThreads=5
 #------------------------------------
 numFileProcessorThreads=2
 
+#------------------------------------
+# Number of threads the BBB server uses to fetch the list of plugins
+# at the beginning of a session (/create). Min=1 means a slower fetch response
+# - will delay the entire list causing slower create response, and therefore,
+# slower client join. Max=half the number of cores for very powerful servers
+#------------------------------------
```

Comment:
```suggestion
#------------------------------------
# Number of threads used to fetch plugin manifest URLs concurrently
#------------------------------------
```

Let's keep it simple, as this config is minor and likely only adjusted by advanced admins. Any detailed explanation can go in the documentation if needed.